packages/python-runner/src/widget.tsx

```diff
@@ -212,7 +214,25 @@ export class PythonFileEditor extends DocumentWidget<
 
     if (this.dockPanel.isEmpty) {
       // Add a tab to dockPanel
-      this.dockPanel.addWidget(this.outputAreaWidget, { mode: 'split-bottom' });
+      const scrollingWidget = new ScrollingWidget({
+        content: this.outputAreaWidget
+      });
+      this.scrollingWidget = scrollingWidget;
+      const scrollUpButton = document.createElement('button');
+      const scrollDownButton = document.createElement('button');
+      scrollUpButton.className = 'elyra-PythonEditor-scrollTop';
+      scrollUpButton.innerHTML = '↑';
+      scrollDownButton.className = 'elyra-PythonEditor-scrollBottom';
+      scrollDownButton.innerHTML = '↓';
+      scrollUpButton.onclick = function() {
```

Comment:
```suggestion
      scrollUpButton.onclick = function(): void {
```
should fix the lint error